samples/sensor/qdec/sample.yaml

```diff
@@ -14,3 +14,18 @@ tests:
         - "Quadrature decoder sensor test"
         - "Position = (.*) degrees"
       fixture: fixture_mech_encoder
+  sample.sensor.nrf_qdec_sensor:
+    tags: sensors
+    platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp
+    integration_platforms:
+      nrf52840dk_nrf52840
+      nrf5340dk_nrf5340_cpuapp
+    timeout: 5
+    harness: console
+    harness_config:
+      type: multi_line
+      ordered: true
+      regex:
+        - "Quadrature decoder sensor test"
+        - "Position = (.*) degrees"
```

Comment:
You can specify most of this (except for `platform_allow` and `integration_platform`) under `common:` to not have to repeat it.